autonomy/cli/replay.py

```diff
@@ -104,7 +105,7 @@ def run_tendermint(build_dir: Path) -> None:
         tendermint_network.stop()
 
 
-def load_docker_config(file_path: Path) -> Dict[str, Any]:
+def load_docker_config(file_path: Path) -> Dict[str, Any]:  # pragma: nocover
     """Load docker config."""
     with open(str(file_path), "r", encoding="utf-8") as fp:
         docker_compose_config = yaml.safe_load(fp)
```

Comment:
`load_docker_config` is patched during tests and this logic is only a wrapper around `yaml.safe_load` so I don't think we need to test this.